### DISCUSSION TITLE
(Mix) Could not start application ssh: could not find application file: ssh.app

### DISCUSSION THREAD

**Discussion by bhavin192** [plausible/analytics#764] [Self-Hosted Support] [answered]
## Prerequisites
- [x] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report
**Describe the bug**
I followed the instructions given in CONTRIBUTING.md to start the environment, but it was giving following error:

     [info] AppSignal disabled.
     [debug] Appsignal.Error.Backend attached to Logger
     [debug] Appsignal.Phoenix.EventHandler attached to [:phoenix, :endpoint, :start]
     [debug] Appsignal.Phoenix.EventHandler attached to [:phoenix, :endpoint, :stop]
     ** (Mix) Could not start application ssh: could not find application file: ssh.app

I had to install the package, erlang-ssh with `sudo dnf install erlang-ssh` to get this working.

**Expected behavior**
The environment should setup after following the instructions.

**Environment (If applicable):**
 - OS: Fedora GNU/Linux

I'm not sure if one has to install this package if they are using some other distribution. If they have to install the package, should the package name present in the `CONTRIBUTING.md`?

**Comment by bhavin192** [comment, marked as answer]
After taking a closer look at the `erlang` package for Alpine, Ubuntu and Fedora as well, it turns out that all of these have `erlang-ssh` as dependency or have the `ssh.app` file in them.

- https://src.fedoraproject.org/rpms/erlang/blob/rawhide/f/erlang.spec#_179
- https://packages.ubuntu.com/focal/erlang
- https://pkgs.alpinelinux.org/contents?file=ssh.app&path=&name=erlang&branch=edge&repo=community&arch=x86_64

**Comment by BlenderDefender** [comment]
I also had this issue on Ubuntu 20.4 and I had to install erlang-ssh seperately.
